packages/aws-rfdk/lib/deadline/lib/render-queue-ref.ts

```diff
@@ -148,9 +151,15 @@ export interface RenderQueueHealthCheckConfiguration {
  * In both cases the certificate chain **must** include only the CA certificates PEM file due to a known limitation in Deadline.
  */
 export interface RenderQueueExternalTLSProps {
+  /**
+   * Whether to enable TLS between the Render Queue and Deadline clients.
+   * @default true
+   */
+  readonly enabled?: boolean;
+
   /**
    * The ACM certificate that will be used for establishing incoming external TLS connections to the RenderQueue.
-   * @default If  not provided then the rfdkCertificate must be provided.
+   * @default If rfdkCertificate and acmCertificate are both not provided, an rfdkCertificate will be generated and used.
```

Comment:
It is obvious but maybe will be worth mention that these certificates will be used or created only when TLS is enabled?

Reply:
Updated